tests/Microsoft.Identity.Test.Unit/pop/PoPTests.cs

```diff
@@ -154,11 +160,14 @@ public async Task POP_WithMissingNonceForPCA_Async()
         {
             using (var httpManager = new MockHttpManager())
             {
-                PublicClientApplication app =
-                    PublicClientApplicationBuilder.Create(TestConstants.ClientId)
-                                                              .WithBrokerPreview()
-                                                              .WithHttpManager(httpManager)
-                                                              .BuildConcrete();
+                
+                var pcaBuilder = PublicClientApplicationBuilder.Create(TestConstants.ClientId)
+                                                              .WithHttpManager(httpManager);
+
+                pcaBuilder = pcaBuilder.WithBroker(
+                    new BrokerOptions(BrokerOptions.OperatingSystems.Windows));
+
+                PublicClientApplication app = pcaBuilder.BuildConcrete();
```

Comment:
Can this class be cleaned up?
```suggestion         
                PublicClientApplication app = PublicClientApplicationBuilder.Create(TestConstants.ClientId)
                                                              .WithHttpManager(httpManager)
                                                              .WithBroker(
                    new BrokerOptions(BrokerOptions.OperatingSystems.Windows))
                                                              .BuildConcrete();
```